intuita-webview/src/main/App.tsx

```diff
@@ -18,6 +20,17 @@ import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import { useTheme } from '../shared/Snippet/useTheme';
 
+const ToastContainerProps = {
```

Comment:
can you rename it to `toastContainerProps` (lowercase)

Reply:
addressed, thanks